tensorflow/core/kernels/multinomial_op.cc

```diff
@@ -163,17 +163,21 @@ class MultinomialOp : public OpKernel {
     const int num_samples = num_samples_t.scalar<int>()();
     OP_REQUIRES(ctx, num_samples >= 0,
                 errors::InvalidArgument(
-                    "num_samples should be nonnegative, got ", num_samples));
-
-    for (int i = 0; i < 2; i++) {
-      const int64_t dim = logits_t.dim_size(i);
-      OP_REQUIRES(ctx, static_cast<int>(dim) == dim,
-                  errors::InvalidArgument(
-                      "logits.shape = ", logits_t.shape().DebugString(),
-                      " too large for int"));
-    }
-    const int batch_size = static_cast<int>(logits_t.dim_size(0));
-    const int num_classes = static_cast<int>(logits_t.dim_size(1));
+                    "num_samples should be non-negative, got ", num_samples));
+
+    const int64_t dim_batch_size = logits_t.dim_size(0);
```

Comment:
nit: why not define batch_size here (doesn't seem there is any benefit to having a dim_batch_size and a batch_size?

(same for the dim_num_classes and num_classes).

Reply:
Good idea, I'll update the variable names.

Reply:
Removed `dim_num_classes` and `dim_batch_size` names.